ios/debugproxy/utun/capture_noop.go

```diff
@@ -0,0 +1,14 @@
+//go:build !darwin
+
+package utun
+
+import (
+	"context"
+	"errors"
+
+	"github.com/danielpaulus/go-ios/ios"
+)
+
+func Live(ctx context.Context, iface string, provider ios.RsdPortProvider, dumpDir string) error {
+	return errors.New("capturing traffic on the utun interface is only supported on MacOS")
```

Comment:
Does this still hold true?

Reply:
Can go-ios sniff tunnels created by itself running on a separate process?

Reply:
it should work on linux as well. however, after changing it to build everywhere the build fails on linux. And I think initially the motivation was that this anyways is mainly used on MacOS I would expect that you always want Xcode/MacOS to perform a certain action that we want to capture